pandora-client-web/src/graphics/def.ts

```diff
@@ -26,42 +37,61 @@ export type ComputedLayerPriority = Exclude<LayerPriority, DoubleOrdered>
 	| `${DoubleOrdered}_BACK`
 	| `${DoubleOrdered}_FRONT`;
 
-export const COMPUTED_LAYER_ORDERING = [
-	'BACKGROUND',
-	'BELOW_BACK_HAIR',
-	'BACK_HAIR',
-
-	'BELOW_ARM_LEFT_BACK',
-	'ARM_LEFT_BACK',
-	'ABOVE_ARM_LEFT_BACK',
-
-	'BELOW_ARM_RIGHT_BACK',
-	'ARM_RIGHT_BACK',
-	'ABOVE_ARM_RIGHT_BACK',
-
-	'BELOW_BODY_SOLES',
-	'BODY_SOLES',
-	'BELOW_BODY',
-	'BODY',
-	'BELOW_BREASTS',
-	'BREASTS',
-	'ABOVE_BODY',
-
-	'BELOW_ARM_LEFT_FRONT',
-	'ARM_LEFT_FRONT',
-	'ABOVE_ARM_LEFT_FRONT',
-
-	'BELOW_ARM_RIGHT_FRONT',
-	'ARM_RIGHT_FRONT',
-	'ABOVE_ARM_RIGHT_FRONT',
-
-	'FRONT_HAIR',
-	'ABOVE_FRONT_HAIR',
-	'OVERLAY',
-] as const satisfies readonly ComputedLayerPriority[];
-
-if (new Set(COMPUTED_LAYER_ORDERING).size !== COMPUTED_LAYER_ORDERING.length || LAYER_PRIORITIES.length + DOUBLE_ORDERED.length !== COMPUTED_LAYER_ORDERING.length) {
-	throw new Error('COMPUTED_LAYER_ORDERING not valid');
+type LeftRight = 'LEFT' | 'RIGHT';
+
+export function useComputedLayerPriority({ view, armsOrder }: Immutable<AppearancePose>): readonly ComputedLayerPriority[] {
+	return useMemo(() => {
+		const arms: [LeftRight, LeftRight] = ['LEFT', 'RIGHT'];
+
+		if (armsOrder.upper === 'right') {
+			arms.reverse();
+		}
```

Comment:
The first arm defined in `arms` will be the one more in the back. That means these should be swapped as "left first" to me means that the left arm is more in the front.